storage/simulation/cluster.go

```diff
@@ -134,13 +216,12 @@ func (c *Cluster) String() string {
 		}
 	}
 
-	var nodeIDs []int
+	var nodeIDs []proto.NodeID
 	for nodeID := range c.nodes {
-		nodeIDs = append(nodeIDs, int(nodeID))
+		nodeIDs = append(nodeIDs, nodeID)
 	}
-	sort.Ints(nodeIDs)
+	sort.Sort(proto.NodeIDSlice(nodeIDs))
```

Comment:
does this actually work correctly? i think you want to declare `nodeIDs` as a `NodeIDSlice` up on L219